### PR TITLE
Working on Documentation

### PR DESCRIPTION
**Pull request by romatik**
Changed some things, but mostly working on documentation

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1 +0,0 @@
-embeddings/*
```
Comment (romatik) [inline .dockerignore:1]: @chrisantonellis, why did you add this in the first place? I tried running in on my home computer and found that this makes it so that you don't have embeddings inside of the container. So I've decided to delete this line and now it works.
This does make local images quite big.

**Comment by romatik** [comment]
@chrisantonellis, until we release the package, I guess we should put all documentation changes in `main` and then push the rendered site to `gh-pages`?

File: .dockerignore (hunk 1 above)
Comment (chrisantonellis) [reply .dockerignore:1]: local image builds take a *long* time when the embeddings are copied to the image on build. if they are mounted at runtime with `docker-compose` the process is a lot faster, however you do not have a complete image locally unless you run with docker-compose. This was to avoid long build times during development but probably has less value now, so safe to remove
